.devops/code-review-pipelines.yml

```
@@ -55,3 +55,12 @@ stages:
               SUBSCRIPTION: PROD-IO-SERVICE-CONN
               ENVIRONMENT: prod
               WORKINGDIR: 'src/core'
+      - job: validate_migration
+        steps:
+          - checkout: self
+          - script: |
+              set -e
+              cd src/psql/selfcare/subscription-migrations
+              echo "Validating migrations"
+              sh ./flyway.sh validate PROD-IO && echo "Validation successful" || echo "Validation failed"; exit 1;
```

```suggestion
              sh ./flyway.sh validate PROD-IO db && echo "Validation successful" || echo "Validation failed"; exit 1;
```
kudos @balanza for the initiative :)